### PR TITLE
[UXE-2130] feat: add created function tracker

### PR DESCRIPTION
**Pull request by aloisio-m-bastian**
### What is the new behavior introduced by this PR?
Add created function event to tracker

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

image

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (fix, feat, test, etc)
- [x] User inputs are sanitized/protected against malicious attacks
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Review by pauloSF0 — APPROVED** [review]

**Review by william-tome — DISMISSED** [review]

**Review by lucasmendes21 — DISMISSED** [review]

File: src/views/EdgeFunctions/CreateView.vue [hunk 1]
```diff
@@ -48,6 +54,12 @@
   })
   const ARGS_INITIAL_STATE = '{}'
 
+  tracker.product.productCreated({
+      productName: 'Edge Function',
```
Comment (matheus-sunderhus) [inline src/views/EdgeFunctions/CreateView.vue:58]: Adicionar esse nome na tipagem de AzionProductsNames para melhorar autocomplete na IDE das opções disponíveis de nomes

**Review by matheus-sunderhus — COMMENTED** [review]
minor changes

File: src/views/EdgeFunctions/CreateView.vue (hunk 1 above)
Comment (aloisio-m-bastian) [reply src/views/EdgeFunctions/CreateView.vue:58]: image

ja está

**Review by HerbertJulio — DISMISSED** [review]
LGTM

**Review by HerbertJulio — APPROVED** [review]

**Review by william-tome — APPROVED** [review]

File: src/views/EdgeFunctions/CreateView.vue (hunk 1 above)
Comment (matheus-sunderhus) [reply src/views/EdgeFunctions/CreateView.vue]: Opa, vlw mestre, olhei na minha dev e não reparei que estava desatualizado. Vlw pelo retorno

**Review by matheus-sunderhus — APPROVED** [review]

**Pull request merged**